libs/category/src/models/category-request.ts

```diff
@@ -3,15 +3,63 @@ import { DaffSortDirectionEnum } from '@daffodil/core/state';
 
 import { DaffCategoryFilterRequest } from './filters/public_api';
 
+/**
+ * When you request a category page, there's two ways to do so, via URI or via ID.
+ */
+export enum DaffCategoryPageRequestKind {
```

Comment:
Should this be `DaffCategoryRequestKind`?

Reply:
I also wonder if we actually need this enum at all. Because we can tell if a request is and id request if it has an ID and we can tell if it is a Url request if it has a URL. By adding `kind` as a required field, we make it so the user needs to add a `kind: DaffCategoryRequestKind.ID` to every action dispatch.

Reply:
> Should this be DaffCategoryRequestKind?

I would be fine with that change.

> we can tell if a request is and id request if it has an ID and we can tell if it is a Url request if it has a URL

I prefer having an enum on which we can switchcase. Most of the users will probably end up using the resolvers anyway.

Reply:
Ok, I think we should go with `DaffCategoryRequestKind`, since it will be used for both category and category page actions

Reply:
changed